.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,35 @@
+name: Test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+#    - name: Lint with flake8
+#      run: |
+#        pip install flake8
+#        # stop the build if there are Python syntax errors or undefined names
+#        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+#        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+#        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
I'd like Flake8 for sure, but I agree on commenting it out and do the specifics when tests are being written.
(Max line length 120 though.)

Reply:
I also think it would be smart to implement a pre-commit hook for flake8 before enabling the flake8 stage. Otherwise I can see this stealing a lot of time.